### ISSUE TITLE
Images in Catalog Products List Widget are Vertically Stretched

### ISSUE THREAD

**Issue by crankycyclops** [magento/magento2#24932]
### Preconditions (*)

Magento version: 2.2.10 & 2.3.x

### Steps to reproduce (*)

1. Upgrade from Magento 2.2.9 -> 2.2.10 or to 2.3.x
2. Clear the cache and reload a page that has a products list widget

### Expected result (*)

Images appear the same as always

### Actual result (*)

Images are stretched vertically

### Additional Notes (*)

I did some digging, and I can see that while Magento 2.2.9 had the following CSS rule:

```
img, object, video, embed {
    height: auto;
    max-width: 100%;
}
```

Magento 2.2.10 has this rule instead:

```
img, object, video, embed {
    max-height: 100%;
    max-width: 100%;
}
```

The rule is defined in both `vendor/magento/magento2-base/lib/web/css/source/lib/_resets.less` on line 54 and `lib/web/css/source/lib/_resets.css` on the same line (filenames are relative to my composer install.)

When I add back the `height: auto;` rule, the images look as they should. I know Magento 2.2.10 is the last release of the 2.2 branch, but I'm reporting this in case it also effects the latest version of 2.3.

**Comment by milindsingh** [comment]
@dhorytskyi 
@sanjay-github 

https://github.com/sanjay-github/magento2/pull/2

**Comment by lewisje** [comment]
I just added the following rule to my frontend theme's CSS file (in the resets section) and the expected behavior was restored (this issue also affected the site logo):

```css
img, object, video, embed {
    height: auto;
}
```

**Comment by webkul-ratnesh** [comment]
@magento i am working on it

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @engcom-Charlie 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-21713` were created

**Issue Available**: @engcom-Charlie, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by webkul-ratnesh** [comment]
This issue is already been fixed in 2.3-develop branch. So I am closing this issue
